tracer/build/_build/docker/gitlab/install_vstudio.ps1

```diff
@@ -0,0 +1,73 @@
+﻿param (
```

Comment:
_Almost_ the same as existing script in agent image, but installs the additional workloads we require (extra ATL components and Windows10SDK.19041)